### PR TITLE
Added software detail pages

### PR DESCRIPTION
**Pull request by ikke09**
Added extra subpages for every software product. For now  I just did this for the first four products in the list. If this first request is getting accepted I will also adding pages for the other products.

The Link of the product opens now these subpages. The page gives a quick overview over the product, so that users can get a first impression of it. Also additional links like help or the donation sides. 

Unfortunately i was not able to test the new pages on my local windows mashine. So hopefully the pages are all working correct.

### CODE REVIEW

File: _software/01_02-gpg4win.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Gpg4win
+
+Gpg4win is a community project and is a suite of encryption tools. Gpg4win tools are based on OpenPGP and focuses not only on emails but also on file encryption.
+
+### Key Facts
+
+* Developer/Publisher: Community
+* Application field: Wide offer on encryption tools 
+* License: Open Source (GNU GPL)
+* Price: All tools are freeware.[Donations](https://www.gpg4win.de/donate.html) are desired.
```
Comment (dschuermann) [inline _software/01_02-gpg4win.md]: "freeware" is a charged term. better use "free of charge"

File: _software/01_02-gpg4win.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* Web: [https://www.gpg4win.de/index.html](https://www.gpg4win.de/index.html
+* Help: Help is given through the communitys
```
Comment (dschuermann) [inline _software/01_02-gpg4win.md]: please use a spell checker: it's "communities"

File: _software/01_03-p≡p.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+modified: 2016-10-25T15:00:00-00:00
+---
+
+{% include base_path %}
+
+{% include toc %}
+
+# p≡p
+
+p≡p is a crossplattform implementation of the OpenPGP Standard. It is available for Android although for Microsoft Outlook. The focus is on a secure connection and an automatic encryption.
```
Comment (dschuermann) [inline _software/01_03-p≡p.md]: wrong use of "although"

File: _software/01_03-p≡p.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+### Key Facts
+
+* Developer/Publisher: p≡p Foundation
+* Application field: Users and companies
+* License: Open Source (GNU GPL)
+* Price: 
+	* Android app: Free (Beta)
+	* Outlook Add-In: about $20
```
Comment (dschuermann) [inline _software/01_03-p≡p.md]: better be precise here

File: _software/01_04-enigmail.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+excerpt: "Email Encryption"
+modified: 2016-10-25T15:00:00-00:00
+---
+
+{% include base_path %}
+
+{% include toc %}
+
+# Enigmail
+Enigmail is an OpenPGP addon for Mozilla Thunderbird. This product brings the encryption features provieded by GnuPG into Thunderbird and allows to sign and encrypt all your emails.
```
Comment (dschuermann) [inline _software/01_04-enigmail.md]: spell-check! "provided"

File: _software/01_03-p≡p.md (hunk 3 above)
Comment (dschuermann) [inline _software/01_03-p≡p.md]: and better use a filename without special characters

**Comment by dschuermann** [comment]
Looks okay to me now. I will merge if this is available for all applications on the email page.

**Comment by dschuermann** [comment]
@ikke09 Can you provide this structure for all email apps? Then we can do this change!

**Comment by ikke09** [comment]
Sorry for the delay. 

Added the details pages for every email application.
I only skipped the email provider because they have no own software products.

Also removed the include statements on every page.

**Comment by dschuermann** [comment]
@ikke09 Merged and fixed in https://github.com/OpenPGP/openpgp.github.io/commit/4558be82b707ad437d769d1905e254512a3287a7

**Pull request merged**